cli/glnexus_cli.cc

```diff
@@ -573,35 +778,32 @@ int main_genotype(int argc, char *argv[]) {
     static struct option long_options[] = {
         {"help", no_argument, 0, 'h'},
         {"residuals", no_argument, 0, 'r'},
-        {"bed", required_argument, 0, 'b'},
```

Comment:
since providing discovered_alleles is not optional to run `glnexus_cli genotype`, I think we should make it a positional argument (immediately following the DB path) instead of an option.